### ISSUE TITLE
train

### ISSUE THREAD

**Issue by mxy519** [NetEase-GameAI/ResNeSt-caffe#3]
Hello, I want to ask  what is the meaning  of caffe environment need permute layer from ssd-caffe ,and if i want to train the network with my own datas what should i do?  Looking forward to your reply,thanks.

**Comment by NetEase-GameAI** [comment]
If you use bvlc-caffe (https://github.com/BVLC/caffe), you need manually add permute layer from ssd-caffe (https://github.com/weiliu89/caffe/tree/ssd), since no Permute layer in bvlc-caffe but ResNeSt need it.
If you wanna train with your data, just do as other networks. For example, change the Input layer to Data layer to use LMDB/LevelDB, and add a SoftmaxWithLoss layer in the end.

**Comment by mxy519** [comment]
Can I use the ssd-caffe directly without using bvlc-caffe?

**Comment by NetEase-GameAI** [comment]
Theoretically yes.

**Comment by mxy519** [comment]
Thank you. Because my classification task is very simple, I only intercepted the part of resnest50 network, and did not use the pre training model during training, resulting in the network not converging and the loss has been very high. What should be done with this?

**Comment by mxy519** [comment]
I cannot download the resnest50 caffemodel, can you share it with baidu cloud

**Comment by NetEase-GameAI** [comment]
> Thank you. Because my classification task is very simple, I only intercepted the part of resnest50 network, and did not use the pre training model during training, resulting in the network not converging and the loss has been very high. What should be done with this?

We did not train resnest models, but only converted them from pytorch to caffe. For simple task, you could finetune pretrained resnest50.

**Comment by NetEase-GameAI** [comment]
> I cannot download the resnest50 caffemodel, can you share it with baidu cloud

We have no Baidu account, but here is a personal link: https://pan.baidu.com/s/13rTjk-q1JJMNj30x3L8vHQ    (code: ucpb)

**Comment by mxy519** [comment]
Thanks. When I finetune the network, the convergence speed of the network is very slow. I want to ask you how to select the parameters when training the network, such as how to set the learning rate, momentum, weight decay coefficient.

**Comment by NetEase-GameAI** [comment]
We have no experience in training ResNeSt. Use the official repo for training help: https://github.com/zhanghang1989/ResNeSt

**Comment by mxy519** [comment]
I want to convert the pretraining model in pytorch(.pth) into Caffe model(.caffemodel). What should I do
